docs/UI_CUSTOMIZATION.md

```diff
@@ -0,0 +1,114 @@
+
+```XML
+<style name="AdyenCheckout" parent="Theme.MaterialComponents(.SOME_EXTENSION)" >
+    <!-- Changes colors and styles to your liking here -->
+</style>
+```
+
+### Option 3: Keep default Adyen theme
+
+You can keep the default Adyen theme by simply adding one line. Optionally, you can change some attributes.
+
+```XML
+<style name="AdyenCheckout" parent="Adyen" >
+    <!-- Optionally: change the primary color and background color of the default theme -->
+    <item name="colorPrimary">#00FF00</item>
+    <item name="android:colorBackground">#FF0000</item>
+</style>
+```
+
+To figure out all the colors and styling you can override have a look at [the Material guides](https://m2.material.io/design/color/the-color-system.html) or check [our Adyen base style](https://github.com/Adyen/adyen-android/blob/main/ui-core/src/main/res/values/styles.xml).
+
+## Customizing the style for specific view types
+
+In case you want to change the styling for a specific view type, you can do this by copying the base styling into your `styles.xml` and change the values as you like. For example, if you want to change the border color of every `TextInputLayout` to red:
+
+```XML
+<style name="AdyenCheckout.TextInputLayout" parent="Widget.MaterialComponents.TextInputLayout.OutlinedBox">
+    <!-- Changes the border color to red -->
+    <item name="boxStrokeColor">#FF0000</item>
+    <!-- These are part of our base style. It's better to leave them in so the UI looks good, but you can change the values to your liking -->
+    <item name="hintTextColor">?attr/colorPrimary</item>
+    <item name="android:minHeight">@dimen/input_layout_height</item>
+</style>
+```
+
+It can be difficult to find which style is applied to which view. To figure this out we recommend to take a look at [our styles.xml](https://github.com/Adyen/adyen-android/blob/main/ui-core/src/main/res/values/styles.xml) or use the Layout Inspector.
+
+## Customizing a specific view
+
+Every view has its own style that builds on top of the view type style. This allows you to customize a specific view. Take for example the card number input field, copy the base style in your `styles.xml`
+
+```XML
+<style name="AdyenCheckout.Card.CardNumberInput" parent="AdyenCheckout.TextInputEditText">
+    <!-- Attribute from base style, it's recommended to leave it as is -->
+    <item name="android:hint">@string/checkout_card_number_hint</item>
+    <!-- Change the background color to magenta and increase the text size -->
+    <item name="android:background">#FF00FF</item>
+    <item name="android:textSize">16sp</item>
+</style>
+```
+
+## Adding dark mode support
+Out of the box the SDK doesn’t support dark mode, but you can easily add this with some additional setup. Copy the base `Adyen` style in your `styles.xml` and change the parent to a Material DayNight theme. Now you can use the `values-night` resource folder to define colors and styles to be used in dark mode.
```

Comment:
Here we say ```Copy the base `Adyen` style``` but in the code sample we use ```style name="AdyenCheckout"```, is that intended?

Reply:
Hmm there are 3 options really, you can override: `Adyen`, `AppTheme` or `AdyenCheckout`. All with the same result. Shall we just only mention `AdyenCheckout`?